packages/serverpod/lib/src/database/table_relation.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:meta/meta.dart';
+import 'package:serverpod/src/database/expressions.dart';
+
+class _TempColumn extends Column<String> {
```

Comment:
It would be nice with a short description of why this is needed here.

Reply:
I opted to remove it.